### ISSUE TITLE
Shift key sticks 

### ISSUE THREAD

**Issue by guraltsev** [DreymaR/BigBagKbdTrixPKL#45]
In the default layout the shift key seems to be sticky. This has some counterintuitive results for example e.g. selecting text using `SHIFT+<arrowkeys>` and then copying it with `CTRL+C` inside a browser window sends the shortcut `CTRL+SHIFT+C` that opens the developer console. 

A partial solution seems to go to Settings and put `stickyTime` much lower. 
![image](https://user-images.githubusercontent.com/7040974/181917526-78b93cfe-34a6-43e0-aefe-6a1925f64a84.png)

While writing this I also noticed the option `stickyMods` from which one can remove `LShift` 
![image](https://user-images.githubusercontent.com/7040974/181917548-6b6547d4-9d65-40f2-b44c-c5a4b3f0125a.png)

This solves the problem!

**Comment by guraltsev** [comment]
P.S. I hope I am not disruptive with these posts. Actually, I encountered these issues and solving them took a bit of time: the options are there but common configuration takes some time. The defaults, like this one, might be initially frustrating because it changes a lot of behavior e.g. copying highlighted text stopped working in browsers, all of a sudden. 

Maybe there is a better place to include these guides or "HowTos". Maybe the Wiki? This would allow the community easy access to contribute such howtos.

**Comment by DreymaR** [comment]
The post aren't disruptive, they discuss interesting points.

I only ever use Extend for shift-selecting these days, so there may be something I'm missing. But if you hold down the Shift key and hit something else, it should not be sticky. The sticky timer should only affect the key when it's tapped. This is on by default since it's so very useful, but you can disable any sticky modifier in the Settings GUI by removing its name from the StickyMods setting.

**Comment by DreymaR** [comment]
Hi again! I can now get back on this issue with good news.

It turns out that any VK-mapped keys that are duplicated on the NumPad (Home/End/PgUp/PgDn/Ins/Del and arrows) don't work as expected with AutoHotkey: When you just send their VK code, the NumPad version is sent instead of the normal one! This is what caused your problems.

In the last EPKL commits and the upcoming release v1.4 though, there are two developments that fix this behavior:
– If the above keys are sent by VK code alone, EPKL intercepts the Send and adds the SC values for the normal version.
– The above keys are now by default mapped with the new SC-type mapping instead, avoiding the problem entirely.
– (Should anyone still want to send the NumPad keys specifically, they can do so with SC### or VK##SC### formalism.)

In conclusion, your problem should be solved now and we've all learnt something weird from it. ` ( のvの) c[_]`
